scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparator.java

```diff
@@ -88,7 +85,8 @@ public double compare(final String baseUseCaseName, final String baseScenarioNam
 		}
 
 		try {
-			int diffColor = new Color(255, 0, 0, 200).getRGB();
+			//int diffColor = new Color(255, 0, 0, 200).getRGB();
+			int diffColor = parameter.getDiffImageColor().getRGB();
```

Comment:
That would be possible. IMHO the configuration should provide a Color (as it is now). It's the Clients decision what he does with the Color. Otherwise the Client needs knowledge about configuration and that it's some magic Hex String...